pkg/registry/apis/folders/sub_children.go

```diff
@@ -46,24 +45,34 @@ func (r *subChildrenREST) NewConnectOptions() (runtime.Object, bool, string) {
 }
 
 func (r *subChildrenREST) Connect(ctx context.Context, name string, opts runtime.Object, responder rest.Responder) (http.Handler, error) {
-	obj, err := r.lister.List(ctx, &internalversion.ListOptions{})
+	obj, err := r.lister.List(ctx, &internalversion.ListOptions{
+		Limit: 500,
+		// TODO, field selector
+	})
```

Comment:
We'll soon have search-after-write on search. At that point we could/should rewrite this using search !?